orchestrator-salt/src/main/resources/salt/salt/cloudera/manager/scripts/setup-autotls.sh

```diff
@@ -16,9 +17,17 @@ source /bin/activate_salt_env
 
 kinit -kt $CM_KEYTAB_FILE $CM_PRINCIPAL
 
-/opt/cloudera/cm-agent/bin/certmanager --location $CERTMANAGER_DIR setup --configure-services $CERTMANAGER_ARGS --override ca_dn="CN=${HOSTNAME}" --stop-at-csr --altname DNS:${HOSTNAME}
+mkdir -p ${CACERTS_DIR}
+
+rm -f ${CACERTS_DIR}/cacerts.p12
+rm -f ${CACERTS_DIR}/cacerts.pem
+
+keytool -importkeystore -srckeystore /usr/lib/jvm/java/jre/lib/security/cacerts -srcstorepass changeit -destkeystore ${CACERTS_DIR}/cacerts.p12 -deststorepass changeit -deststoretype PKCS12
```

Comment:
are you sure that this java path is always valid? or we don't bother with custom images/other OS-s now?

Reply:
Thanks, fixed to JAVA_HOME